src/pop_verifier.erl

```diff
@@ -287,12 +292,14 @@ handle_info({custom_timer_tick, CurrentTime}, State) ->
 
     {noreply, State2};
 
-handle_info(Data, _State) ->
-    erlang:error(unexpected_handle_info, [Data]).
-
 handle_cast(Data, _State) ->
     erlang:error(unexpected_handle_cast, [Data]).
 
+% Kind of hacky - forward messages as cast calls
```

Comment:
:O

Reply:
yeah, too lazy to refactor my unit test code; it was written before I figured out gen_server thing

Reply:
idk, seems legit to me.